app/services/solidus_bolt/users/add_payment_source_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module SolidusBolt
+  module Users
+    class AddPaymentSourceService < SolidusBolt::BaseService
```

Comment:
Does it add the payment or make in sync solidus payment sources with bolt payment methods?

Reply:
renamed in 57d29313dff9bbab7d779c4fe648e1aacfa41b3e